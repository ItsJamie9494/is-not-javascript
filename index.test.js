const renderer = require("react-test-renderer");
const { isNotJavascript, useIsNotJavascript } = require("./index");

test("Is Not Javascript", () => {
  expect(isNotJavascript()).toBe(false);
});

const TestComponent = () => {
  const notJS = useIsNotJavascript();

  return <p>{notJS ? "Is Not Javascript" : "Is Javascript"}</p>;
};

test("Use Is Not Javascript", () => {
  const component = renderer.create(<TestComponent />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
