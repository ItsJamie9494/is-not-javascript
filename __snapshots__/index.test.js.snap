// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Use Is Not Javascript 1`] = `
<p>
  Is Javascript
</p>
`;
